src/main/scala/encry/view/EncryNodeViewHolder.scala

```diff
@@ -318,8 +319,10 @@ class EncryNodeViewHolder[StateType <: EncryState[StateType]](auxHistoryHolder:
 
   def txModify(tx: Transaction): Unit = nodeView.mempool.put(tx) match {
     case Success(newPool) =>
+      //TODO for what we update wallet?
       updateNodeView(updatedVault = Some(nodeView.wallet), updatedMempool = Some(newPool))
       nodeViewSynchronizer ! SuccessfulTransaction[EncryProposition, Transaction](tx)
+      nodeView.mempool.putElementToBloomFilter(tx.id)
```

Comment:
Maybe it would be better to call this method inside `mempool.put`?

Reply:
.put provides another logic